easybuild/easyconfigs/s/structure/structure-2.3.4-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,34 @@
+#
+# This file is an EasyBuild reciPy by Christian Meesters (2017)
+# HPC-group University of Mainz (https://hpc.uni-mainz.de)
+
+easyblock = 'MakeCp'
+
+name = 'structure'
+version = '2.3.4'
+
+homepage = 'https://web.stanford.edu/group/pritchardlab/structure.html'
+description = """The program structure is a free software package for using multi-locus
+ genotype data to investigate population structure. Its uses include infering 
+ the presence of distinct populations, assigning individuals to populations, 
+ studying hybrid zones, identifying migrants and admixed individuals, and estimating
+ population allele frequencies in situations where many individuals are migrants or admixed. 
+ It can be applied to most of the commonly-used genetic markers, including SNPS, 
+ microsatellites, RFLPs and AFLPs."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
```

Comment:
@cmeesters As I mentioned, if you use `GCCcore` as toolchain, you need to include `binutils` as a build dependency.

Also, please remove the other easyconfigs, should work with `eb --update-pr 5202 :structure-2.3.4.eb`